src/unstract/adapters/x2text/unstructured_community/src/unstructured_community.py

```diff
@@ -41,11 +42,14 @@ def process(
         input_file_path: str,
         output_file_path: Optional[str] = None,
         **kwargs: dict[Any, Any],
-    ) -> str:
-        output: str = UnstructuredHelper.process_document(
+    ) -> TextExtractionResult:
+        extracted_text: str = UnstructuredHelper.process_document(
             self.config, input_file_path, output_file_path
         )
-        return output
+
+        return TextExtractionResult(
+            extracted_text=extracted_text, extraction_metadata=None
```

Comment:
Why it return None? Can we do this default value in dto ?

Reply:
for certain adapters, we won't have any meta data, unlike LLMWhisperer.

Reply:
Addressed.